src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -212,6 +218,31 @@
      */
     public static final RegistryObject<Codec<NoneStructureModifier>> NONE_STRUCTURE_MODIFIER_TYPE = STRUCTURE_MODIFIER_SERIALIZERS.register("none", () -> Codec.unit(NoneStructureModifier.INSTANCE));
 
+    /**
+     * Stock holder set type that represents any/all values in a registry. Can be used in a holderset object with { "type": "forge:any" }
```

Comment:
The JSON code should be wrapped in a `{@code}` tag. (Same comment for each of the below javadocs.)

```suggestion
     * Stock holder set type that represents any/all values in a registry. Can be used in a holderset object with {@code { "type": "forge:any" }}.
```